sdk/python/src/dagger/telemetry/attributes.py

```diff
@@ -1,4 +0,0 @@
-from typing import Final
-
-UI_ENCAPSULATE: Final = "dagger.io/ui.encapsulate"
```

Comment:
This used to be used by the two root spans in the SDK, at the beginning of the _registration_ and _invocation_ phases for a function call, but those were removed, so just removing this before someone depends on it.